sdk.yaml

```diff
@@ -271,6 +271,9 @@ Realtime:
             not necessarily the id for this client.
     Publish:
       .specification: RTL6
+      .synopsis: |
+        Send a message on this channel.
+        This is a message over the Realtime connection, not a REST operation.
```

Comment:
Isn't it clear, by way of being under the `Realtime` field, that this is a realtime publish, not a REST one?

Reply:
Yes, you're completely right. I guess what I was trying to articulate was that this is implemented as a Realtime protocol message over the websocket connection, rather than via REST. But, I agree, there's no reason why I need to be explicit about that for this node when I've not done the same for others. I shall remove this sentence. Thanks.

Reply:
https://github.com/ably/features/pull/28/commits/354d4191fe57ba6760567ce36610c32bc7ca8de6